zqd/core.go

```diff
@@ -36,6 +38,20 @@ type Core struct {
 	SortLimit int
 	taskCount int64
 	logger    *zap.Logger
+
+	// spaceOpsLock protects the spaceOps map and the active and
+	// deletePending fields inside the spaceOpsState's.
+	spaceOpsLock sync.Mutex
```

Comment:
Nit: Call this spaceOpsMu for consistency with existing sync.Mutex/RWMutex names.